react/Header/SignInRegister/SignInRegister.js

```diff
@@ -19,7 +16,7 @@ export default function SignInRegister({ linkRenderer, returnUrl }) {
       {
         linkRenderer({
           'data-analytics': 'sign-in',
-          href: `/Login/Standalone${urlSuffix}`,
+          href: appendReturnUrl('/Login/Standalone', returnUrl),
```

Comment:
@dborysiewicz should we be updating the sign in & register url's to the new pages yet?